Common/Constants/include/CommonConstants/make_pdg_header.py

```diff
@@ -127,6 +130,10 @@ class Pdg(Enum):
     kXiCCPlusPlus = 4422
     kXiCPlus = 4232
     kXiC0 = 4132
+    kXiCPlus3055 = 4325
+    kXiCPlus3080 = 4326
+    kXiCZero3055 = 4315
+    kXiCZero3080 = 4316
```

Comment:
Please try to stay consistent with the naming style in `TPDGCode.h`, e.g. see:
`kRho770_0`
`kRho770Plus`
`kRho3_1690_0`
`kA2_1320Plus`